src/utils/progress.js

```diff
@@ -0,0 +1,15 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { RULES } from "../constants/messages.js";
+
+const progress = () => {
```

Comment:
```suggestion
const shouldAdvance = () => {
```
`progress`는 boolean 값을 반환하기 때문에 `is`, `has`, `should`와 같은 접두사를 붙여서 `shouldAdvance`와 같이 수정 해줘도 좋을 것 같아요!